From 476e0fd451a95e87f7e6cca61c53592e495060ac Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Fri, 10 Aug 2012 15:31:26 +0800
Subject: ENGR00220038-1 build config: update default build config

update default build config, build back ground overlay driver based
on ipu device driver instead of the former prp_view finder driver.

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 181cf31..495cdf7 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,6 +1,6 @@
 #
 # Automatically generated make config: don't edit
-# Linux/arm 3.0.15 Kernel Configuration
+# Linux/arm 3.0.35 Kernel Configuration
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -314,6 +314,7 @@ CONFIG_MACH_MX6Q_SABREAUTO=y
 # MX6 Options:
 #
 # CONFIG_IMX_PCIE is not set
+CONFIG_USB_EHCI_ARC_H1=y
 # CONFIG_MX6_INTER_LDO_BYPASS is not set
 CONFIG_ISP1504_MXC=y
 # CONFIG_MXC_IRQ_PRIOR is not set
@@ -1622,7 +1623,7 @@ CONFIG_MXC_CAMERA_OV5642=m
 CONFIG_MXC_TVIN_ADV7180=m
 CONFIG_MXC_CAMERA_OV5640_MIPI=m
 CONFIG_MXC_CAMERA_SENSOR_CLK=m
-CONFIG_MXC_IPU_PRP_VF_SDC=m
+CONFIG_MXC_IPU_DEVICE_QUEUE_SDC=m
 CONFIG_MXC_IPU_PRP_ENC=m
 CONFIG_MXC_IPU_CSI_ENC=m
 CONFIG_VIDEO_MXC_OUTPUT=y
@@ -1938,7 +1939,6 @@ CONFIG_USB_OTG=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_ARC=y
 CONFIG_USB_EHCI_ARC_OTG=y
-CONFIG_USB_EHCI_ARC_H1=y
 # CONFIG_USB_EHCI_ARC_HSIC is not set
 # CONFIG_USB_STATIC_IRAM is not set
 CONFIG_USB_EHCI_ROOT_HUB_TT=y
-- 
1.7.9.5

